flight/Modules/Attitude/attitude.c

```diff
@@ -1410,7 +1406,6 @@ static void updateNedAccel()
 	accel_ned[2] += GRAVITY;
 	
 	NedAccelData accelData;
-	NedAccelGet(&accelData);
```

Comment:
Yeah, I'm thinking of the UAVO fields changing in future though, it'll be quite fragile.